fenix/app/src/androidTest/java/org/mozilla/fenix/ui/SettingsAdvancedTest.kt

```diff
@@ -50,6 +50,7 @@ class SettingsAdvancedTest {
         mockWebServer.shutdown()
     }
 
+    // TestRail link: https://testrail.stage.mozaws.net/index.php?/cases/view/2092699
     // Walks through settings menu and sub-menus to ensure all items are present
     @Test
     fun settingsAdvancedItemsTest() {
```

Comment:
Maybe rename this as the manual test to "verifyAdvancedSettingsSectionItemsTest"

Reply:
Sounds good !